packages/core/src/fhir-deduplication/shared.ts

```diff
@@ -235,6 +234,38 @@ export function pickMostDescriptiveStatus<T extends string>(
   return status;
 }
 
-export function isBlacklistedText(concept: CodeableConcept | undefined): boolean {
-  return concept?.text?.toLowerCase().includes("no known") ?? false;
+export function hasBlacklistedText(concept: CodeableConcept | undefined): boolean {
+  const knownCodings = concept?.coding?.filter(c => !isUnknownCoding(c));
+  return (
+    concept?.text?.toLowerCase().includes(NO_KNOWN_SUBSTRING) ??
+    Boolean(knownCodings?.length) ??
```

Comment:
this change adding 

`Boolean(knownCodings?.length)`

causes us to fail 25 tests. What its doing is saying anything with a known coding is true and then it blacklists it which is the opposite of what we want. 

Lets get rid of the check entirely or change it to 

`    !knownCodings?.length`

Reply:
see here: https://github.com/metriport/metriport/pull/2699/files#r1741374075